cli/flags.rs

```diff
@@ -406,6 +406,12 @@ pub fn flags_from_vec(
       }
     }
     ("xeval", Some(eval_match)) => {
+      flags.allow_net = true;
+      flags.allow_env = true;
+      flags.allow_run = true;
+      flags.allow_read = true;
+      flags.allow_write = true;
+      flags.allow_high_precision = true;
```

Comment:
Can you mention this in the "deno help xeval" docs?

Reply:
Done, I also added same notice to REPL and eval subcommands